### PR TITLE
CLI should handle long and multi-line property values better.

### PR DESCRIPTION
**Pull request by techcobweb**
# Why ?
In order to cope with some .md markdown text we need the set/get properties to work well with properties which have newlines inside.

- super-long values can be set, but when displayed in summary view they get cropped with a `...(cropped)` string at the end. Newlines are replaced with `\n`.
- raw format shows new lines replaced with `\n`
- yaml format creates decent yaml, and accepts yaml with multiple-line input such as this:
```
piVersion: galasa-dev/v1alpha1
kind: GalasaProperty
metadata:
    namespace: mcobbett6
    name: markdown.test.1
data:
    value: |-
        # This is the title
        This is a line of text in the this is a longer description
        which spans multiple lines and 

        And happends to be very long.

        Here is an example of a list
        - one
        - two
        - three
        
        And a [link here](https://galasa.dev)
```

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-274-vj65r for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-274-kw8qc for details. If you are unable to do so, please contact a member of the Galasa team.

File: pkg/propertiesformatter/yamlFormatter.go [hunk 1]
```diff
@@ -47,6 +47,8 @@ func (*PropertyYamlFormatter) FormatProperties(cpsProperties []galasaapi.GalasaP
 		yamlRepresentationBytes, err = yaml.Marshal(property)
 		if err == nil {
 			yamlStr := string(yamlRepresentationBytes)
+
+			// TODO: Can anyone please explain why we do this substitution ? Wworried that if the value the user supplies has 'apiversion' inside it will get corrupted...
 			yamlStr = strings.ReplaceAll(yamlStr, "apiversion", "apiVersion")
```
Comment (eamansour) [inline pkg/propertiesformatter/yamlFormatter.go:52]: We can remove this

**Review by eamansour — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-274-jrmnc for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**